packages/jazz-tools/src/implementation/createContext.ts

```diff
@@ -15,6 +15,10 @@ export type AuthResult =
     | {
           type: "existing";
           credentials: { accountID: ID<Account>; secret: AgentSecret };
+          saveCredentials?: (credentials: {
```

Comment:
Seems like a good hook for saving to localStorage, but it's not on the FixedCreds... putting in as an optional